charts/sentence-collector/values.yaml

```diff
@@ -0,0 +1,60 @@
+replicaCount: 1
+
+image:
+  repository: mozilla/sentencecollector
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "2.0.9"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: "sentence-collector"
+
+externalSecret:
+  create: false
+
+serviceAccount:
+  create: true
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext:
+  fsGroup: 2000
+
+securityContext:
+  runAsNonRoot: true
+  runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+  hosts:
+    - host: chart-example.local
+      paths: []
+
+resources:
+  limits:
+    cpu: 500m
+    memory: 512Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+autoscaling:
+  enabled: false
+
+mysql:
+  enabled: true
+  fullnameOverride: mysql
+  mysqlUser: sc
```

Comment:
Where is this database user provisioned?  Is there an additional step out of review that this occurs (I probably just missed it)?

Reply:
Great question. The user is actually provisioned using that line.

There's a Chart dependency to MySQL defined [here](https://github.com/mozilla-it/helm-charts/pull/40/files#diff-635c66a55e493048aeb13e00b0452675R18-R21). In this block we enable (install) the dependency. For example in prod we will use RDS, this is really only for end to end test.
The MySQL Chart define those attributes as a way to create the user, password etc... [check here](https://github.com/helm/charts/blob/master/stable/mysql/README.md).
Finally, we pass a secret to the application referrencing the user created with the pw [here](https://github.com/mozilla-it/helm-charts/pull/40/files#diff-e3caff692f39406f0683f8702c0f6d40R17)

Reply:
That makes sense.  Thanks fo the detailed responses, it is appreciated.